File: Pidev_EGOV/Pi_Egov_Java/src/esprit/egov/entities/Passeport.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package esprit.egov.entities;

import java.sql.Date;



/**
 *
 * @author boumaiza
 */
public class Passeport {
    private int numPasseport;
    private Date dateDelivrance;
    private Date dateValidite;
    private String couleurYeux;
    private String validation;
    private int numCinUtilisateur;

    public Passeport() {
    }

    public Passeport(int numPasseport, Date dateDelivrance, Date dateValidite, String couleurYeux, String validation, int numCinUtilisateur) {
        this.numPasseport = numPasseport;
        this.dateDelivrance = dateDelivrance;
        this.dateValidite = dateValidite;
        this.couleurYeux = couleurYeux;
        this.validation = validation;
        this.numCinUtilisateur = numCinUtilisateur;
    }

    public Passeport(Date dateDelivrance, Date dateValidite, String couleurYeux, String validation, int numCinUtilisateur) {
        this.dateDelivrance = dateDelivrance;
        this.dateValidite = dateValidite;
        this.couleurYeux = couleurYeux;
        this.validation = validation;
        this.numCinUtilisateur = numCinUtilisateur;
    }

    public int getNumPasseport() {
        return numPasseport;
    }

    public void setNumPasseport(int numPasseport) {
        this.numPasseport = numPasseport;
    }

    public Date getDateDelivrance() {
        return dateDelivrance;
    }

    public void setDateDelivrance(Date dateDelivrance) {
        this.dateDelivrance = dateDelivrance;
    }

    public Date getDateValidite() {
        return dateValidite;
    }

    public void setDateValidite(Date dateValidite) {
        this.dateValidite = dateValidite;
    }

    public String getCouleurYeux() {
        return couleurYeux;
    }

    public void setCouleurYeux(String couleurYeux) {
        this.couleurYeux = couleurYeux;
    }

    public String getValidation() {
        return validation;
    }

    public void setValidation(String validation) {
        this.validation = validation;
    }

    public int getNumCinUtilisateur() {
        return numCinUtilisateur;
    }

    public void setNumCinUtilisateur(int numCinUtilisateur) {
        this.numCinUtilisateur = numCinUtilisateur;
    }

    @Override
    public String toString() {
        return "Passeport{" + "numPasseport=" + numPasseport + ", dateDelivrance=" + dateDelivrance + ", dateValidite=" + dateValidite + ", couleurYeux=" + couleurYeux + ", validation=" + validation + ", numCinUtilisateur=" + numCinUtilisateur + '}';
    }
    
    
    
    
    
}

File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/Passeport.php
<?php

namespace MyApp\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Passeport
 *
 * @ORM\Table(name="passeport", uniqueConstraints={@ORM\UniqueConstraint(name="num_passeport", columns={"numPasseport"})}, indexes={@ORM\Index(name="numCinUtilisateur", columns={"numCinUtilisateur"})})
 * @ORM\Entity
 */
class Passeport
{
    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateDelivrance", type="date", nullable=false)
     */
    private $datedelivrance;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateValidite", type="date", nullable=false)
     */
    private $datevalidite;

    /**
     * @var string
     *
     * @ORM\Column(name="couleurYeux", type="string", length=10, nullable=true)
     */
    private $couleuryeux;

    /**
     * @var string
     *
     * @ORM\Column(name="validation", type="string", length=10, nullable=true)
     */
    private $validation;

    /**
     * @var integer
     *
     * @ORM\Column(name="numPasseport", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $numpasseport;

    /**
     * @var \MyApp\UserBundle\Entity\Utilisateur
     *
     * @ORM\ManyToOne(targetEntity="MyApp\UserBundle\Entity\Utilisateur")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="numCinUtilisateur", referencedColumnName="id_uti")
     * })
     */
    private $numcinutilisateur;



    /**
     * Set datedelivrance
     *
     * @param \DateTime $datedelivrance
     * @return Passeport
     */
    public function setDatedelivrance($datedelivrance)
    {
        $this->datedelivrance = $datedelivrance;

        return $this;
    }

    /**
     * Get datedelivrance
     *
     * @return \DateTime 
     */
    public function getDatedelivrance()
    {
        return $this->datedelivrance;
    }

    /**
     * Set datevalidite
     *
     * @param \DateTime $datevalidite
     * @return Passeport
     */
    public function setDatevalidite($datevalidite)
    {
        $this->datevalidite = $datevalidite;

        return $this;
    }

    /**
     * Get datevalidite
     *
     * @return \DateTime 
     */
    public function getDatevalidite()
    {
        return $this->datevalidite;
    }

    /**
     * Set couleuryeux
     *
     * @param string $couleuryeux
     * @return Passeport
     */
    public function setCouleuryeux($couleuryeux)
    {
        $this->couleuryeux = $couleuryeux;

        return $this;
    }

    /**
     * Get couleuryeux
     *
     * @return string 
     */
    public function getCouleuryeux()
    {
        return $this->couleuryeux;
    }

    /**
     * Set validation
     *
     * @param string $validation
     * @return Passeport
     */
    public function setValidation($validation)
    {
        $this->validation = $validation;

        return $this;
    }

    /**
     * Get validation
     *
     * @return string 
     */
    public function getValidation()
    {
        return $this->validation;
    }

    /**
     * Get numpasseport
     *
     * @return integer 
     */
    public function getNumpasseport()
    {
        return $this->numpasseport;
    }

    /**
     * Set numcinutilisateur
     *
     * @param \MyApp\UserBundle\Entity\Utilisateur $numcinutilisateur
     * @return Passeport
     */
    public function setNumcinutilisateur(\MyApp\UserBundle\Entity\Utilisateur $numcinutilisateur = null)
    {
        $this->numcinutilisateur = $numcinutilisateur;

        return $this;
    }

    /**
     * Get numcinutilisateur
     *
     * @return \MyApp\UserBundle\Entity\Utilisateur 
     */
    public function getNumcinutilisateur()
    {
        return $this->numcinutilisateur;
    }
}
